snafu/utils/get_prometheus_data.py

```diff
@@ -48,42 +49,44 @@ def get_all_metrics(self):
         # check get_data bool, if false by-pass all processing
         if self.get_data:
             start_time = time.time()
+            include_file_dir = "/opt/snafu/snafu/utils/prometheus_labels/"
```

Comment:
This expression will enforce snafu to be available in that directory. It might be situations where it's not there.
Could we replace it by something more dynamic? e.g.
```suggestion
            base_dir = os.path.dirname(os.path.abspath(__file__)))
            include_file_dir = os.path.join(base_dir, "utils", "prometheus_labels")
```

Reply:
You are right will update